modules/updating-control-plane-only-update.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/control-plane-only-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-control-plane-only-update_{context}"]
+= Control Plane Only update
+
+The following procedure pauses all non-master machine config pools and performs updates from {product-title} <4.y> to <4.y+1> to <4.y+2>, then unpauses the previously paused machine config pools.
```

Comment:
Is it possible to only put part of this phrase in backticks?
```suggestion
The following procedure pauses all non-`master` machine config pools and performs updates from {product-title} <4.y> to <4.y+1> to <4.y+2>, then unpauses the previously paused machine config pools.
```

If so, should probably do that, since I think the only time we can use "master" in docs is if it's in reference to a hardcoded value in our software that we can't change. And if it's a software element, it should probably use backticks